### DISCUSSION TITLE
不能跨wireguard收发文件

### DISCUSSION THREAD

**Discussion by cndeng** [localsend/localsend#611] [L-简体中文]
跨 wireguard 无法发现客户端

**Comment by mm10079** [comment]
可以，我使用wireguard傳送成功
起初也是無法發現客戶端，手動添加防火牆後就可以使用了

**Reply by oyster-zzz** [reply]
请问我跨wireguard传送，当设备不在一个网段时无法直接发现设备，您有遇到吗？如何解决呢

**Comment by lieTAT** [comment]
> 跨 wireguard 无法发现客户端

windows的话,故障排除那边点下自动修复就好了,然后就能互相发现传文件了
